data/blog/kafka-monitoring-opentelemetry.mdx

```diff
@@ -204,9 +204,9 @@ This use case is designed to cater scenarios where platform owners want to get a
 
 **Approach**
 
-In this proposal, we plan to have two tabs listing producer and consumer services grouped by topics. A user would be able to see details about producer and consumer services, and on clicking a particular Topic & Service group, they can see a side panel that will show more detailed RED metrics for each partition. 
+In this proposal, we plan to have two tabs listing producer and consumer services grouped by topics. A user would be able to see details about producer and consumer services, by clicking on a particular topic and service group, and accessing a side panel displaying more detailed RED (Rate, Error, Duration) metrics for each partition.
```

Comment:
`by clicking on a particular topic and service group, and accessing a side panel`
 
 this is not accurate. only when they click a point in the graph they see the side panel. Side panel is not always accessible.

Reply:
done